src/basic_recipes/stairs.jl

```diff
@@ -62,24 +62,6 @@ function plot!(p::Stairs{<:Tuple{<:AbstractVector{<:Point2}}})
             error("Invalid step $step. Valid options are :pre, :post and :center")
         end
     end
-
-    if isa(p.color[], AbstractVector)
-        stepcolor = lift(p.color, p.step) do color, step
-            if step == :pre
-                c = [color';color'][2:end]
-            elseif step == :post
-                c = [color';color'][1:end-1]
-            elseif step == :center
-                c = [color';color'][1:end]
-            else
-                error("Invalid step $step. Valid options are :pre, :post and :center")
-            end
-            c
-        end
-        lines!(p, steppoints; color=stepcolor, [x for x in pairs(p.attributes) if x[1] != :step && x[1] != :color]...)
-    else
-        lines!(p, steppoints; [x for x in pairs(p.attributes) if x[1] != :step]...)
-    end
-    p
```

Comment:
strange - looks like the merge commit took this code out.  Was that intentional @SimonDanisch?  (Was going through the PR backlog and saw this).

Reply:
Not sure to be honest...